MangaManager/src/MetadataManager/CoverManager/CoverManager.py

```diff
@@ -141,6 +172,9 @@ def __init__(self, master, super_: GUIApp = None, **kwargs):
         self.title(self.__class__.name)
         if super_ is not None:
             self._super = super_
+        global overlay_image
```

Comment:
Try to avoid using global

Reply:
That's one of the optimizations. could pass it as a method parameter when instantiating each ComicFrame. Doesn't really look the cleanest to me.

Reply:
How does this optimization work or rather, what is the goal of this global?

Reply:
I can't just create a PhotoImage. A Tk() instance has to be created prior. So that's why i update the variable once it's called from the ui (so tk is already instantiated (tkinter stuff)).
Open to suggestions.

Reply:
Correct, you need to hold a reference to the image so it doesn't get dereferenced. Is this class instance only used when the window is open? If so, then I would leave at class level.

However, there may be opportunity to move the Photo image to the Resource class which can hold onto images for us by introducing a get_image(key, cache=True)

Reply:
There has to be some way but it can't be too early.